src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/*
+ * This file is part of the jolicode/elastically library.
+ *
+ * (c) JoliCode <coucou@jolicode.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JoliCode\Elastically\Bridge\Symfony\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
+
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder('elastically');
+        $rootNode = $treeBuilder->getRootNode();
+
+        $rootNode
+            ->children()
+                ->arrayNode('connections')
```

Comment:
Is it possible to get a minimum of 1 connection ? I'm not sure it's checked atm

Reply:
I don't see why we should force at least one connection. The bundle must work as expected with or without connection. ATM, the bundle only provide DI for connection, but later it might do other things (profiler, logger, command, etc). So we must not enforce this constraint.